src/datasets/filesystems/__init__.py

```diff
@@ -0,0 +1,34 @@
+import importlib
+
+import fsspec
+
+
+_has_s3fs = importlib.util.find_spec("s3fs")
```

Comment:
```suggestion
_has_s3fs = importlib.util.find_spec("s3fs") is not None
```
(nit) to have a boolean